helm-charts/splunk-otel-collector/templates/config/_otel-k8s-cluster-receiver-config.tpl

```diff
@@ -73,6 +73,7 @@ receivers:
   {{- end }}
 
 processors:
+  {{- include "splunk-otel-collector.k8sClusterReceiverAttributesProcessor" . | nindent 2 }}
```

Comment:
Lets refactor this a little for a couple reasons. This configurations will always be included even if not used and we could relocate it to be closer to similar configurations like it.
- Move this content to around lines 92-112
- Add conditional clauses appropriately to make this configuration content only included when it is used. Something like this.

```
  {{- if and $clusterReceiver.eventsEnabled (eq (include "splunk-otel-collector.logsEnabled" .) "true") }}
  {{- include "splunk-otel-collector.k8sClusterReceiverAttributesProcessor" . | nindent 2 }}
  {{- end }}`
```

Reply:
done